packages/router/src/globals.ts

```diff
@@ -0,0 +1,14 @@
+import type React from 'react'
+
+import type { Router } from './router'
+
+declare global {
+  interface Window {
+    __TSR__ROUTER__: Router
```

Comment:
This is something I forgot to rename when I was studying Tanstack router. I'd call this `__TUONO_ROUTER__` and also change the property setter down below